apps/src/applab/applab.js

```diff
@@ -826,14 +826,25 @@ Applab.exportApp = function(expoOpts) {
   var html = document.getElementById('divApplab').outerHTML;
   studioApp().resetButtonClick();
 
-  const {mode, expoSnackId} = expoOpts || {};
+  // TODO: find another way to get this info that doesn't rely on globals.
+  const appName =
+    (window.dashboard && window.dashboard.project.getCurrentName()) || 'my-app';
```

Comment:
applab.js already imports `project`:

https://github.com/code-dot-org/code-dot-org/blob/30e54e5c54f96ab13e2d94d6891ac4cc363ad537/apps/src/applab/applab.js#L66

Which is the same module as `window.dashboard.project`:

https://github.com/code-dot-org/code-dot-org/blob/1cad5ac667f9ed1a3aa31063e6d06bd21a78e2bd/apps/src/sites/studio/pages/code-studio.js#L63

So I think you can just

```js
const appName = project.getCurrentName() || 'my-app';
```